e2e-tests/test/debug-apis.test.ts

```diff
@@ -72,4 +73,45 @@ describe("debug namespace", function () {
     const [output_number] = primary.interface.decodeFunctionResult("calculate", output);
     expect(output_number.toNumber()).to.equal(12);
   });
+
+  it("Should return null if txn hash is unknown", async function () {
```

Comment:
We should wrap these `it(...)` tests inside another `describe("debug_traceTransaction")` to group together tests by the endpoint they're validating